keyboards/pteron56/keymaps/via/keymap.c

```diff
@@ -0,0 +1,64 @@
+/*pteron56
+*Copyright (C) 2023 @kraken-jokes
+*
+*This program is free software: you can redistribute it and/or modify
+*it under the terms of the GNU General Public License as published by
+*the Free Software Foundation, either version 3 of the License, or
+*(at your option) any later version.
+*
+*This program is distributed in the hope that it will be useful,
+*but WITHOUT ANY WARRANTY; without even the implied warranty of
+*IMERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*GNU General Public License for more details.
+*
+*You should have received a copy of the GNU General Public License
+*along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
+#define _COPY LCTL(KC_C)
+#define _L1_GUI LT(1,KC_LGUI)
+#define _L1_DEL LT(1,KC_DEL)
+#define _L2_APP LT(2,KC_APP)
+#define _L2_ALT LT(2,KC_LALT)
+#define _PRTSC LGUI(LSFT(KC_S))
+#define _CUT LCTL(KC_X)
+#define _PASTE LCTL(KC_V)
+#define _ALTF4 LALT(KC_F4)
+#define _ALT_ESC LALT(KC_ESC)
+#define _UNDO LCTL(KC_Z)
+#define _CTL_TAB LCTL(KC_TAB)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,   KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, 
+        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,   KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_EQL, 
+        KC_BSPC, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,   KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, 
+        KC_SPC,  KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,   KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT, 
+                          _L1_GUI, KC_LCTL, KC_LSFT, _L2_ALT, _L1_DEL, KC_SPC, _L2_APP, KC_TAB
```

Comment:
removed unused customer keycodes
```suggestion
#define _L1_GUI LT(1,KC_LGUI)
#define _L1_DEL LT(1,KC_DEL)
#define _L2_APP LT(2,KC_APP)
#define _L2_ALT LT(2,KC_LALT)

const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    [0] = LAYOUT(
        KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,   KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, 
        KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,   KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_EQL, 
        KC_BSPC, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,   KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, 
        KC_SPC,  KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,   KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_ENT, 
                          L1_GUI, KC_LCTL, KC_LSFT, L2_ALT, L1_DEL, KC_SPC, L2_APP, KC_TAB
```